modules/claims_api/app/swagger/claims_api/v1/swagger.json

```diff
@@ -1,8 +1,9 @@
 {
   "openapi": "3.0.1",
   "info": {
-    "title": "API V1",
+    "title": "Benefits Claims",
     "version": "v1",
+    "termsOfService": "https://developer.va.gov/terms-of-service",
```

Comment:
I noticed this is something that Appeals includes in their [rwag config](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/spec/support/rswag_config.rb#L16). Figured it doesn't hurt to add them to ours, as well